cmd/buildah/rmi.go

```diff
@@ -39,14 +43,18 @@ var (
 func rmiCmd(c *cli.Context) error {
 	force := c.Bool("force")
 	removeAll := c.Bool("all")
+	removeDangling := c.Bool("prune")
```

Comment:
Just to be consistent and to make it easier to find, can you change removeDangling to pruneDangling throughout?  I'll try to mark them.